Source/Particles/Gather/FieldGather.H

```diff
@@ -430,4 +430,87 @@ void doGatherShapeN(const GetParticlePosition& getPosition,
         );
 }
 
+/**
+ * \brief Field gather for a single particle
+ *
+ * /param xp, yp, zp                : Particle position coordinates
```

Comment:
serious <3 for this doc string!

Just a little typo:
```suggestion
 * \param xp, yp, zp                : Particle position coordinates
```

Reply:
Thanks, fixed.